reporting/lsrd_stats.py

```diff
@@ -1,675 +1,388 @@
-# 005               05-31-2013      Adam Dosch      Making order separation by source so we can
-#                                                   tell the difference between which interface
-#                                                   someone used to submit order (ee or espa)
-# 006               06-03-2013      Adam Dosch      Adding os.environ['JAVA_HOME'] to top-level
-#                                                   to fix environment sourcing issue from cron
-# 007               08-21-2013      Adam Dosch      Adding '__author__' references
-#                                                   Updated email_from to FQDN host
-# 008               11-25-2013      Adam Dosch      Removing GLS collection and UI reporting from
-#                                                   the scripts --- we be tearing that shit OUT.
-#                                                   I will keep logic in pig scripts, but just not
-#                                                   report those in the e-mail output
-# 009               01-30-2014      Adam Dosch      Updating 'resultsondemandledaps' to 'resultsondemand'
-#                                                   as the outdir name has changed in lsrd_ondemand_metrics
-#                                                   pig script
-# 010               05-01-2014      Adam Dosch      Cleaning up dotfile temp cred mess and db_connect fuction
-# 011               09-08-2014      Adam Dosch      Updating unique users query to use a substring index
-#                                                   since our email field does not get populated in the
-#                                                   DB anymore since 2.5.0 release
-# 012               11-07-2014      Adam Dosch      Updated queries to distinguish USGS vs. non-USGS better 
-#                                                   and added to distribution list
-# 013		         03-31-2015      Adam Dosch      Updating notification adddresses
-# 014               09-04-15        David Hill      Minimum changes necessary to port to postgres db
-#
-#########################################################################################
-
-__author__ = "Adam Dosch"
-
-import os
-import commands
-import sys
 import re
 import datetime
+import calendar
 import argparse
-import smtplib
-import socket
+import os
+import subprocess
 import traceback
+import getpass
+
+from dbconnect import DBConnect
+from utils import get_cfg, send_email
+
+
+FILE_PATH = os.path.realpath(__file__)
+LOG_FILE = '/data/logs/espa.cr.usgs.gov-access_log.1'
+
+# This info should come from a config file
+EMAIL_FROM = 'espa@espa.cr.usgs.gov'
+# EMAIL_TO = ['jenkerson@usgs.gov', 'lowen@usgs.gov', 'dhill@usgs.gov', 'klsmith@usgs.gov']
+EMAIL_TO = ['klsmith@usgs.gov']
+DEBUG_EMAIL_TO = ['klsmith@usgs.gov']
+EMAIL_SUBJECT = 'LSRD ESPA Metrics for {0} to {1}'
+
+ORDER_SOURCES = ('EE', 'ESPA')
+
+
+def arg_parser():
```

Comment:
I found something really cool a while back that Morton is using, haven't used it myself yet.  Check this out: http://click.pocoo.org/5/arguments/